index.html

```diff
@@ -1,33 +1,31 @@
-<!DOCTYPE HTML>
-
-<html>
-	<head>
-		<title>Minimo Blog Challenge</title>
-		<meta charset="utf-8" />
-		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-		<link rel="stylesheet" href="css/normalize.css" />
-		<link href="https://fonts.googleapis.com/css?family=Playfair+Display|Ubuntu" rel="stylesheet">
-		<link rel="stylesheet" href="css/styles.css" />
-		<script src="js/picturefill.min.js" async></script>
-	</head>
-	<body>
-		<header>
-				<img class="minimo" src="images/Minimo.jpg" alt="Minimo logo">
-			<div class="main-nav">
-				<ul>
-					<li><a href="#lifestyle">lifestyle</a></li>
-					<li><a href="#photodiary">photodiary</a></li>
-					<li><a href="#music">music</a></li>
-					<li><a href="#travel">travel</a></li>
-				</ul>
-			</div>
-			<div>
-				<picture>
-					<source srcset="images/Header_photo.jpg 1059w, images/Header_photo_rectangle.jpg 778w, images/Header_photo_square.jpg 429w" sizes="90vw" />
-					<img src="images/Header_photo.jpg" alt="picture of two women surrounded by sand" />
-				</picture>
-			</div>
-		</header>
-	</body>
-</html>
-
+<!DOCTYPE HTML>
+
+<html>
+	<head>
+		<title>Minimo Blog Challenge</title>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="css/normalize.css" />
+		<link href="https://fonts.googleapis.com/css?family=Playfair+Display|Ubuntu" rel="stylesheet">
+		<link rel="stylesheet" href="css/styles.css" />
+	</head>
+	<body>
+		<div class="container">
+			<header>
+					<img class="minimo" src="images/Minimo.jpg" alt="Minimo logo">
+				<div class="main-nav">
+					<ul>
+						<li><a href="#lifestyle">lifestyle</a></li>
+						<li><a href="#photodiary">photodiary</a></li>
+						<li><a href="#music">music</a></li>
+						<li><a href="#travel">travel</a></li>
+					</ul>
+				</div>
+				<div class="header-photo">
+					<img src="images/Header_photo.jpg" />
+				</div>

```

Comment:
Jeżeli wyniesiesz kontener ze zdjęciem poza `header`w ten sposób:
```
<header>..</header>
<div class="header-photo"></div>
```
i dasz dla <header> style `overflow: hidden;` to wtedy background zadziała.

Zrób tak a zobaczysz, że działa :) Tak będzie lepiej ze względu na to, że semantycznie będzie lepiej bo zdjęcie akurat nie powinno należeć do headera.

A czemu `overflow: hidden;`? Poczytaj o floatach jak się zachowują. Można ten problem też rozwiązać właściwością`clear`. 

https://css-tricks.com/all-about-floats/